bin/start_aws.sh

```diff
@@ -167,16 +173,18 @@ echo "Checking for required secrets"
 if pulumi config get sirius:accounts_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius >/dev/null 2>&1; then
   echo "Password found for the sirius accounts database"
 else
-  echo "Create a password for the sirius accounts database"
-  pulumi config set --secret sirius:accounts_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius
+  ACCOUNTS_PW=$(createpw)
+  pulumi config set --secret sirius:accounts_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius $ACCOUNTS_PW
+  echo "Created password for the sirius accounts database"
```

Comment:
If we are not exposing the passwords to the user, is it necessary to let them know about it?  If I'm told it was created I may start to think I need access to it.  If we think there's value, perhaps the message (or maybe a "script done summary" type message at the end) could advise more specifically around accessing the secrets to directly access the app databases.

Reply:
This makes sense; these bash scripts will be going away at some point but for now I'm going to make a change at a high level that removes some of these extraneous messages (which I think I put in way back when I was first writing it as kind of notes to myself). 

I'm going to look at putting an overall "here's what I did" at the bottom of the scripts.